static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -316,6 +315,9 @@ const initialiseBanner = (): void => {
     ];
 
     initBannerPicker(bannerList);
+    // } else {
```

Comment:
These three lines are safe to be removed. I'm guessing they are a product of resolving conflict after the rebase.